pkg/test/utils.go

```diff
@@ -96,7 +96,7 @@ func CompareGoldenFile(t *testing.T, p string, got string, normalizers ...func(s
 		if err := os.WriteFile(p, []byte(got), 0644); err != nil {
 			t.Fatalf("failed to write golden output %s: %v", p, err)
 		}
-		t.Errorf("wrote updated golden output to %s", p)
+		t.Logf("wrote updated golden output to %s", p)
```

Comment:
I think I have come around to the ergonomics here. Still it would be nice to have a little different or stronger signal and I can explore that separately.